Comp2022/src/main/cpp/subsystems/Drivetrain.cpp

```diff
@@ -669,7 +669,7 @@ void Drivetrain::MoveWithLimelightExecute(void)
     double ty = robotContainer->m_vision.GetVertOffsetDeg();
     bool tv = robotContainer->m_vision.GetTargetValid();
 
-    if (tv == false)
+    if (tv == false && m_limelightDebug >= 1)
     {
         VelocityArcadeDrive(0, 0);

```

Comment:
This VelocityArcadeDrive line here can't be hidden behind the `if m_limelightDebug >= 1` - we must drive(0,0) regardless of whether we want that printout